lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -19,6 +19,7 @@
 
 from edxmako.shortcuts import render_to_response
 from courseware.courses import get_course_with_access
+from openedx.core.djangoapps.content.course_overviews.utils import is_discussion_enabled
```

Comment:
The `is_discussion_enabled` method should belong somewhere in `django_comment_client`.  Possibly in https://github.com/edx/edx-platform/blob/master/lms/djangoapps/django_comment_client/utils.py.